### PR TITLE
1137 devex - Address feedback

### PR DESCRIPTION
**Pull request by ujahio**
[Trello Card](https://trello.com/c/bHHK51Jg/1137-reconsider-rehome-stateaddeddocketnumbers)
 - Address feedback (bug) that was incorrectly resetting text and icon to "Add docket numbers to the caption" after editing the title of the order in the create order flow. Editing the title of the order should never change the text of the button to select cases.
 - Created cypress test to validate UI elements during the creation of the order.

### CODE REVIEW

**Review by TomElliottFlexion — APPROVED** [review]

File: cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import { loginAsDocketClerk } from '../../helpers/auth/login-as-helpers';
+
+describe('Docket clerk creates and edits draft order with selected docket numbers', function () {
+  const leadCase = '111-19';
+
+  beforeEach(() => {
+    loginAsDocketClerk();
```
Comment (codyseibert) [inline cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts:7]: I would remove this.  I see you loginAsDocketClerk, but then later on at the start of the it statement you loginAsDocketClerk again

File: cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      .invoke('attr', 'data-metadata')
+      .then(val => {
+        expect(val).to.equal(consolidatedCases);
+      });
+  });
+
+  it('should edit the draft order with newly selected cases', () => {
+    let draftsCount: number = 0;
+
+    loginAsDocketClerk();
```
Comment (codyseibert) [inline cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts:93]: here, you've already logged in in the beforeEach(), I'd personally just remove the before each as it requires keeping more stuff in our head to understand how the test works

File: cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  });
+
+  it('should create an order with ALL cases selected', () => {
+    const orderTitle = 'Order first title';
+    const orderEventCode = 'O';
+    let consolidatedCases: string = '';
+    let draftsCount: number = 0;
+
+    cy.get('[data-testid="docket-number-search-input"]').clear();
```
Comment (codyseibert) [inline cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts:16]: I'd check if there is already a helper for doing this (I think there is), if not, I can see this type of thing used many places and would be a great helper utility

**Review by codyseibert — APPROVED** [review]

File: cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts (hunk 1 above)
Comment (ujahio) [reply cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts:7]: That makes sense. I have made the adjustment: [97a2a01](https://github.com/ustaxcourt/ef-cms/pull/4317/commits/97a2a010fe4defa750614cbdfeafbbc8bb3a09c2)

File: cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts (hunk 2 above)
Comment (ujahio) [reply cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts:93]: I agree and have made the adjustment. [97a2a01](https://github.com/ustaxcourt/ef-cms/pull/4317/commits/97a2a010fe4defa750614cbdfeafbbc8bb3a09c2)

File: cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts (hunk 3 above)
Comment (ujahio) [reply cypress/cypress-integration/integration/docket-clerk-multi-docket-draft-order.cy.ts:16]: There is a helper for that actually and I am going to use it. Thanks..

**Pull request merged**